test/commands/InspectCommandTest.ts

```diff
@@ -0,0 +1,152 @@
+import { createSandbox, SinonSandbox } from "sinon";
+import { expect } from "chai";
+import { Collection, EmbedField, GuildMember, GuildMemberRoleManager, Message, Role } from "discord.js";
+import {BaseMocks, CustomMocks} from "@lambocreeper/mock-discord.js";
```

Comment:
Make sure that you use spaces {  }
It also seems like CustomMocks is not being used